### DISCUSSION TITLE
Random mod doesn't work in playlists

### DISCUSSION THREAD

**Discussion by 2minwia** [ppy/osu#14911] [Q&A] [answered]
made a playlist with a few songs in osu!standard mode, added the random mod to all of them, then i tried to play the songs, and .. they just get stuck on the loading screen, giving an error message about an unhandled exception.

[runtime.log](https://github.com/ppy/osu/files/7261357/runtime.log)

reproduce by:
* make a playlist lobby with a song (or just join the one i made called "what is this, a SRT?" because it seems like you cant delete playlists)
* give song the random mod
* join the lobby, try to play the song

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/14912
